packages/contracts/src/typechain-types/factories/Endpoint__factory.ts

```diff
@@ -2,8 +2,7 @@
 /* tslint:disable */
 /* eslint-disable */
 
-import { Contract, Signer, utils } from "ethers";
-import type { Provider } from "@ethersproject/providers";
+import { Contract, Interface, type ContractRunner } from "ethers";
```

Comment:
@frankfka curious why did the typechains got regenerated? to be compatible w/ v6?